industrial_trajectory_filters/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 2.8.3)
 
 project(industrial_trajectory_filters)
 
+add_compile_options(-std=c++11)
```

Comment:
I changed all versions to `2.8.12`. Where do you find the minimum required version for a CMake option?